tests/components/tuya/test_climate.py

```diff
@@ -11,12 +11,16 @@
 from homeassistant.components.tuya import ManagerCompat
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ServiceNotSupported
 from homeassistant.helpers import entity_registry as er
 
 from . import DEVICE_MOCKS, initialize_entry
 
 from tests.common import MockConfigEntry, snapshot_platform
 
+WINDSPEED_LOW = 1
+WINDSPEED_HIGH = 2
+
```

Comment:
I don't think using constants is necessary here
I think the existing values are sufficient
```suggestion
```